person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    @age > 18 || @parent_permission == true
```

Comment:
- [ ] Under private methods you have defined `of_age?`. Kindly use it here instead of `@age > 18`
- [ ] There is redundancy with the snippet `@parent_permission == true` here. `@parent_permission` has already been initialized to `true` so there is really no need to check for that since it will always be true anyways. Kindly remove the `== true` at the end of the line.

Reply:
thanks for the review 🐱 changes have been made ♻️